app/v2/controllers/requestParsers/validators/RetrieveUkPropertyBusinessAnnualSummaryValidator.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v2.controllers.requestParsers.validators
+
+import config.AppConfig
+
+import javax.inject.{Inject, Singleton}
+import v2.controllers.requestParsers.validators.validations.{NinoValidation, TaxYearValidation}
+import v2.models.errors.MtdError
+import v2.models.request.retrieveUkPropertyBusinessAnnualSummary.RetrieveUkPropertyBusinessAnnualSummaryRawData
+
+@Singleton
+class RetrieveUkPropertyBusinessAnnualSummaryValidator @Inject()(appConfig: AppConfig) extends Validator[RetrieveUkPropertyBusinessAnnualSummaryRawData] {
+
+  private lazy val minTaxYear = appConfig.minimumTaxV2Uk
```

Comment:
This is set to "2022" in config. You will need to add to add new configuration parameters for the min and max tax years for historic endpoints, and then to use those here.